CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # Change Log
 
+## [0.22.0](https://github.com/eclipse/lemminx/milestone/35?closed=1) (October 19, 2022)
+
+### Enhancements
+
+ * Initialize RelaxNG support with validation/completion/hover. See [#828](https://github.com/eclipse/lemminx/issues/828).
+ * Support `xml.format.closingBracketNewLine` setting with experimental formatter. See [#1247](https://github.com/eclipse/lemminx/issues/1247).
+ * Support `xml.format.xsiSchemaLocationSplit` setting with experimental formatter. See [#1246](https://github.com/eclipse/lemminx/issues/1246).
+ * Support `xml.format.spaceBeforeEmptyCloseTag` setting with experimental formatter. See [#1245](https://github.com/eclipse/lemminx/issues/1245).
+ * Support `xml.format.joinContentLines` setting with experimental formatter. See [#1244](https://github.com/eclipse/lemminx/issues/1244).
+ * Support `xml.format.joinCommentLines` setting with experimental formatter. See [#1243](https://github.com/eclipse/lemminx/issues/1243).
+ * Support `xml.format.preserveEmptyContent` setting with experimental formatter. See [#1242](https://github.com/eclipse/lemminx/issues/1242).
+ * Support `xml.format.joinCDATALines` setting with experimental formatter. See [#1241](https://github.com/eclipse/lemminx/issues/1241).
+ * Support `xml.format.preservedNewlines` setting with experimental formatter. See [#1240](https://github.com/eclipse/lemminx/issues/1240).
+ * Support `xml.format.enforceQuoteStyle` setting with experimental formatter. See [#1239](https://github.com/eclipse/lemminx/issues/1239).
+ * Support `xml.format.emptyElements` setting with experimental formatter. See [#1238](https://github.com/eclipse/lemminx/issues/1238).
+ * Provide comment formatting for experimental formatter. See [#1259](https://github.com/eclipse/lemminx/issues/1259). 
+ * CodeAction for cvc-complex-type.2.4.b: "insert all expected" vs. "insert all possible". See [#1255](https://github.com/eclipse/lemminx/issues/1255).  
+ * Suppress validation kinds based on file pattern. See [#1275](https://github.com/eclipse/lemminx/issues/1275).
+ * Report only XML syntax error for *.exsd files. See [#1308](https://github.com/eclipse/lemminx/pull/1308).  
+ * XML validation errors of "Language Servers" are only partially visible. See [#1145](https://github.com/eclipse/lemminx/issues/1145).
+ * Validate uri of XML catalog. See [#823](https://github.com/eclipse/lemminx/issues/823).
+ * Do not complete paths in attr unless beginning of value looks like a path. See [#1293](https://github.com/eclipse/lemminx/pull/1293). 
+ * Use the same proxy configuration for HTTPS. See [#1253](https://github.com/eclipse/lemminx/pull/1253).
```

Comment:
Use the HTTP proxy configuration for HTTPS as well

Reply:
fixed